lib/presentation/widgets/todo_list.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:hive/hive.dart';
+
+import '../../model/todo.dart';
+
+class TodoList extends StatefulWidget {
+  const TodoList({Key? key}) : super(key: key);
+
+  @override
+  _TodoListState createState() => _TodoListState();
+}
+
+class _TodoListState extends State<TodoList> {
+  @override
+  Widget build(BuildContext context) {
+    final todoBox = Hive.box('todos');
```

Comment:
why is was used like that, not initialized in initState for example? 
Also, you can get the listenable of Hive to the builder as it was shown in documentation here https://github.com/hivedb/hive#hive-%EF%B8%8F-flutter

Reply:
You can initiate this todoBox as variable with late modifier:
`late Box todoBox;
initState {
   todoBox = Hive.box
}`